static/src/javascripts/projects/common/modules/commercial/third-party-tags/outbrain.js

```diff
@@ -60,41 +62,35 @@ define([
                 }
             };
 
-            widgetCodeImage = widgetConfig[breakpoint].image[getSection()];
+            widgetCodeImage = widgetConfig[breakpoint].image[section];
             widgetCode = widgetCodeImage;
 
             fastdom.write(function () {
                 $outbrain.css('display', 'block');
                 $container.append($.create(template(outbrainTpl, { widgetCode: widgetCode })));
-            });
-
-            if (breakpoint !== 'mobile') {
-                widgetCodeText  = widgetConfig[breakpoint].text[getSection()];
 
-                fastdom.write(function () {
+                if (breakpoint !== 'mobile') {
+                    widgetCodeText  = widgetConfig[breakpoint].text[section];
                     $container.append($.create(template(outbrainTpl, { widgetCode: widgetCodeText })));
-                });
-            }
+                }
 
-            return require(['js!' + outbrainUrl]);
-        }
-    }
+                require(['js!' + outbrainUrl]);
+            });
+        },
 
-    function getSection() {
-        return config.page.section.toLowerCase().match('news')
-            || _.contains(['politics', 'world', 'business', 'commentisfree'], config.page.section.toLowerCase()) ? 'sections' : 'all';
-    }
+        getSection: function () {
+            return config.page.section.toLowerCase().match('news')
+                || _.contains(['politics', 'world', 'business', 'commentisfree'], config.page.section.toLowerCase()) ? 'sections' : 'all';
+        },
 
-    function init() {
-        mediator.on('modules:commercial:dfp:rendered', function (event) {
-            if (event.slot.getSlotId().getDomId() === 'dfp-ad--merchandising-high'
-                && event.isEmpty && config.page.section !== 'childrens-books-site') {
-                load();
+        init: function () {
+            if (config.switches.outbrain && !config.page.isPreview && !identity.isUserLoggedIn() && config.page.section !== 'childrens-books-site') {
```

Comment:
That is taken care of in [back end](https://github.com/guardian/frontend/blob/099006dc1a5badeff939a876f2261217551d720a/common/app/views/fragments/contentFooter.scala.html)